apps/passport-client/src/dispatch.ts

```diff
@@ -497,6 +505,12 @@ async function loadAfterLogin(
     storage.storage,
     await getPackages()
   );
+  const validationErrors = validatePCDCollection(pcds);
```

Comment:
I'd save this for later so you can validate it against the User object fetched below too.

Reply:
To clarify ambiguous words: I meant wait to validate later, not save the validationErrors for later.

Reply:
I'm not quire sure that makes sense, as this validation would prevent an invalid upload from occurring, whereas if I save it for later the invalid state would clobber valid state on the server-side.

Reply:
More ambiguous words to clarify.  By "later" I meant line 523 below, immediately after fetching the user object.  The difference being that once a user object is available, it can be included in the validation, letting you validate that there is a matching identity PCD.

Reply:
Also note that this state just came from the server during login, so I see this validation as being more about "incoming" bad data than "outgoing" bad data.  It's certainly true that once state is invalid we don't want it uploaded, though.

Reply:
Cool, I think I ended up doing the thing you suggested. The validation now waits until the user has been downloaded.